source/performance/F0.Analyzers.Benchmarks/CodeAnalysis/CodeFixes/DeclareRecordClassExplicitlyBenchmarks.cs

```diff
@@ -17,11 +17,11 @@ public DeclareRecordClassExplicitlyBenchmarks()
 	[GlobalSetup]
 	public void Setup()
 	{
-		var code =
-@"using System;
+		var code = """
```

Comment:
Converted all interpolated and/or verbatim strings that comprise _C# text_ to _raw string literals_.